src/svg2luxor.jl

```diff
@@ -254,11 +325,15 @@ function pathsvg(svg)
     fsize = get_current_setting().fontsize
     xdoc = parse_string(svg)
     xroot = root(xdoc)
-    def_element = get_elements_by_tagname(xroot, "defs")[1]
-    # create a dict for all the definitions
     defs = Dict{String,Any}()
-    for def in collect(child_elements(def_element))
-        defs[attribute(def, "id")] = def
+    try
```

Comment:
For each try catch I would like to know why it can fail and maybe there is an option to avoid the try catch then.

Reply:
This try statement checks if we have any `def` tags , if they exist we add the defs to  `defs` dictionary
if there are no `def` tags ( like in the case of dvisvgm which draws directly and doesnt predefine 
paths to use) `get_elements_by_tagname(xroot, "defs")` will return an empty iterable and cause a bounds error .

Reply:
Yeah! i probably shouldn't discard all errors , should `throw(e)` if its not a `BoundsError`